packages/web-service/src/pages/work-activity/work-payment-exempt-reason/work-payment-exempt-reason.js

```diff
@@ -24,6 +23,8 @@ const {
 export const getData = async request => {
   const { applicationId } = await request.cache().getData()
   const pageData = await request.cache().getPageData()
+  const payload = pageData?.payload || {}
```

Comment:
If this necessary? pageData is plumbed into page automatically by page-handler.js and it should usually just be referenced directly in the templates. Look in the cache.